source/mail-access.rst

```diff
@@ -59,19 +59,19 @@ We recommend using IMAP over POP to access your email. By default, POP will pull
 SMTP
 ----
 
-+--------------------+----------------------------------------------+
-|Server              | :term:`your Hostname`                        |
-+--------------------+----------------------------------------------+
-|Port                | ``587``                                      |
-+--------------------+----------------------------------------------+
-|SSL/TLS Encryption  | STARTTLS                                     |
-+--------------------+----------------------------------------------+
-|Username            | Your email address, including the domain     |
-+--------------------+----------------------------------------------+
-|Password            | Your password for the email address          |
-+--------------------+----------------------------------------------+
-
-.. warning:: Don't use ports ``25`` or ``465`` for sending mails! Unfortunately many clients try to use these ports, this won't work on Uberspace.
++--------------------+---------------------------------------------+
+|Server              | :term:`your Hostname`                       |
++--------------------+----------------------+----------------------+
+|Port                | ``587``              | ``456``              |
```

Comment:
I think this should be `465` not `456`.

Reply:
Thanks for catching this. Resolved in https://github.com/Uberspace/manual/issues/351